src/main/resources/application.yml

```
@@ -1 +1,14 @@
+    name: otboo
+  profiles:
+    active: dev
+
+logging:
+  level:
+    root: info
+
+springdoc:
+  swagger-ui:
+    enabled: true
+    path: /swagger-custom-ui.html
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)